src/compo/tempo_nc2ioda.py

```diff
@@ -121,6 +124,7 @@ def _read(self):
                 sfp = np.ma.array(sfp, mask=mask)
                 ak = ncd.groups['support_data'].variables['surface_pressure'].Eta_A
                 bk = ncd.groups['support_data'].variables['surface_pressure'].Eta_B
+                print(np.shape(mask))
```

Comment:
is this print still needed, if so a little more description would likely make it even more helpful ;0)

Reply:
no it was just for testing, removing it now